#!/usr/bin/env python
PACKAGE = "raw_image_pipeline_white_balance"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Lu_offset",    double_t,    0, "Lu offset", 0,  -2.000, 2.000)
gen.add("Lv_offset",    double_t,    0, "Lv offset", 0,  -2.000, 2.000)
gen.add("uv0",    double_t,    0, "uv0", -1.421875,  -10.0, 10.0)
gen.add("bright_thr",  double_t,    0, "Bright pixels saturation threshold", 1.0,  0.0, 1.0)
gen.add("dark_thr",  double_t,    0, "Dark pixels saturation threshold", 0.0,  0.0, 1.0)

exit(gen.generate(PACKAGE, "raw_image_pipeline_white_balance", "RawImagePipelineWhiteBalance"))